input/metadata/default_metadata.json

```
@@ -2,140 +2,140 @@
+      "description": "Energy prices used in the EEE module.",
       "path": "input/data/EEE/default_costs.csv",
       "type": "csv",
       "properties": "economic_properties"
     },
     "emission": {
       "title": "Emissions data",
-      "description": "The configuration file for the EEE module that is related to emissions.",
+      "description": "Tillage emission values used in the EEE module.",
       "path": "input/data/EEE/default_emissions.csv",
       "type": "csv",
       "properties": "emissions_properties"
     },
     "purchased_feeds_emissions": {
       "title": "Emissions from purchased feeds",
-      "description": "Specifies how much emissions are associated with purchased feed types.",
+      "description": "Purchased feeds emission values used in the EEE module.",
       "path": "input/data/EEE/full_feeds_emissions.csv",
       "type": "csv",
       "properties": "feed_emissions_properties"
     },
     "feed": {
       "title": "Feed data",
-      "description": "The configuration file for the feed module.",
+      "description": "Feeds available for each animal combination, purchased feeds and their prices, feed storage options, and user-defined ration percentages and associated parameters.",
       "path": "input/data/feed/default_feed.json",
       "type": "json",
       "properties": "feed_properties"
     },
     "NRC_Comp": {
       "title": "NRC Comp data",
-      "description": "Nutrients!",
+      "description": "Nutritional information for each feed, following NASEM (2021) guidelines.",
       "path": "input/data/feed/NRC_comp.csv",
       "type": "csv",
       "properties": "NRC_Comp_properties"
     },
     "NASEM_Comp": {
       "title": "NASEM Comp data",
-      "description": "Nutrients!",
+      "description": "Nutritional information for each feed, following NRC (2001) guidelines.",
       "path": "input/data/feed/NASEM_Comp_with_TDN.csv",
       "type": "csv",
       "properties": "NASEM_Comp_properties"
     },
     "manure_management": {
       "title": "Manure data",
-      "description": "The configuration file for the manure module.",
+      "description": "Manure management decisions, including bedding, separator, and treatment information.",
       "path": "input/data/manure/manure_management.json",
       "type": "json",
       "properties": "manure_management_properties"
     },
     "soil_1": {
       "title": "Soil data",
-      "description": "The first configuration file for the soil module.",
+      "description": "Characteristic soil information, including composition, slope, and layer details.",
       "path": "input/data/soil/default_soil.json",
       "type": "json",
       "properties": "soil_profile_properties"
     },
     "soil_2": {
-      "title": "Soil data",
-      "description": "The second configuration file for the soil module.",
+      "title": "Soil data #TODO does this need a unique name for the future?",
```

```suggestion
      "title": "Soil data",
```
No, uniqueness is just a nice-to-have, and only benefits the readability of the metadata.